ERCS/erc-7578.md

```diff
@@ -0,0 +1,275 @@
+
+* `[LEGAL OWNER OF UNDERLYING PHYSICAL ASSET]` **Identification of legal owner of underlying physical asset:** [NAME], [UNIQUE IDENTIFICATION NUMBER] OR [NETWORK ID]. 
+<br> *Comment: This information identifies the first point of contact for the physical redemption of the underlying physical asset and defines the contractual party to the NFT holder to legally enforce the rights conferred to the NFT holder. Same as with the NFT issuer’s identity, the legal owner is identified by a name but may also be identifiable via unique identification number or network ID that is issued by a service provider that stores relevant information about the legal owner.*
+
+* `[STORAGE LOCATION]` **Identification of storage location of underlying physical asset:** [PHYSICAL ADDRESS OR JURISDICTION]. 
+<br> *Comment: The storage location provides the NFT holder with a second point of contact for the physical redemption and could also be used to demonstrate adequate storage of a particular physical assets that require special storage conditions.*  
+
+* `[LEGAL CONTRACT]` **Type of legal relationship (with link to document IPFS):**
+<br> *Comment: This provides the NFT holder with the legal basis of the relationship between them and the legal owner of the underlying physical asset. Ideally the information is provided by embedding a link to the actual legal documentation. The more information is accessible via the NFT, the better the NFT holder is able to assess the legal risks associated with enforcement of physical redemption of the underlying physical asset.*
+
+* `[APPLICABLE LAW]` **Governing Law and Jurisdiction:** [JURISDICTION]
+<br> *Comment: This information allows the NFT holder to quick access the legal risk associated to the disclosed jurisdiction without the need of going through the entire legal documentation.*
+
+* `[DECLARED VALUE]` **Value of the underlying asset:** [VALUE] provides a value for the underlying physical asset. 
+<br> *Comment: The declared value provides an indication for the NFT holder regarding the underlying asset’s value and allows auxiliary services to tie certain functions to it. The declared value of the underlying physical asset does not necessarily represent the market value.*  
+
+## Specification
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+[ERC-721](./eip-721.md) compliant contracts MAY implement this EIP.
+
+`Properties` MUST be set before a token can be minted.
+
+The `tokenId` MUST be greater than 0.
+
+The `terms` parameter MUST be a link to a document that is stored on IPFS.
```

Comment:
This in underspecified. Should `terms` be a [CID](https://docs.ipfs.tech/concepts/content-addressing/)? An `ipfs://...` URL?

Reply:
updated to be more specific